pom.xml

```diff
@@ -276,6 +276,12 @@
                     <version>2.8.1</version>
                     <configuration>
                         <argLine>${argLine} -Djava.awt.headless=true -noverify</argLine>
+                        <includes>
+                            <include>**/*.java</include>
```

Comment:
Spoke with @rzwiefel about this. It's confounding. Not clear how it doesn't now skip all Spock tests; not clear how it _fails_ if he includes `**/*.groovy` as an include pattern.

@lessarderic do you have some insight? It's not like the groovyc compiler is generating java files as an interim step...